package.json

```diff
@@ -41,5 +41,5 @@
     "rimraf": "^2.5.4",
     "safe-publish-latest": "^1.1.1"
   },
-  "version": "2.0.2"
+  "version": "2.1.0"
```

Comment:
I would not bump the version in a feature pull request.

Reply:
Yeah I was trying to figure out how this was usually handled? Will the version be updated later make the npm package available?